src/test/java/org/solid/testharness/config/UserCredentialsTest.java

```diff
@@ -23,15 +24,16 @@
 
     @Test
     public void createFromModel() throws IOException {
-        final Resource user = iri("http://example.org/alice");
+        final Resource user = iri(TestData.SAMPLE_NS, "alice");
         final Reader reader = new StringReader(
+                "@prefix ex: <" + TestData.SAMPLE_NS + "> .\n" +
```

Comment:
I see this is problematic for having RDF as external files, but perhaps it could be done by using relative URIs that are resolved in the parsing?

Reply:
Does the comment above resolve this?

Reply:
Yup!